asterius/src/Asterius/JSRun/NonMain.hs

```diff
@@ -18,60 +31,62 @@ linkNonMain store_m extra_syms = (m, link_report)
     (_, m, link_report) =
       linkModules
         LinkTask
-          { progName = ""
-          , linkOutput = ""
-          , linkObjs = []
-          , linkLibs = []
-          , linkModule = mempty
-          , Asterius.Ld.debug = False
-          , Asterius.Ld.gcSections = True
-          , Asterius.Ld.binaryen = False
-          , Asterius.Ld.verboseErr = True
-          , Asterius.Ld.outputIR = Nothing
-          , rootSymbols = extra_syms
-          , Asterius.Ld.exportFunctions = []
+          { progName = "",
+            linkOutput = "",
+            linkObjs = [],
+            linkLibs = [],
+            linkModule = mempty,
+            Asterius.Ld.debug = False,
+            Asterius.Ld.gcSections = True,
+            Asterius.Ld.binaryen = False,
+            Asterius.Ld.verboseErr = True,
+            Asterius.Ld.outputIR = Nothing,
+            rootSymbols = extra_syms,
+            Asterius.Ld.exportFunctions = []
           }
```

Comment:
Sigh, "ormolu" doesn't seem to be configurable and I prefer the aligned-comma style :'(